src/odb/src/codeGenerator/helper.py

```diff
@@ -99,6 +99,14 @@ def is_bit_fields(field, structs):
             return True
     return False
 
+def get_plural_name(name):
```

Comment:
I remember this same issue in another db from a past life.  English is weird enough that there is a library just for this purpose `inflect`.  This will probably hold us for a while.

Reply:
Yes I came across that library but I didn't want to add a new python library to the build.